modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,58 @@
+/*This file is part of OpenCV project.
+It is subject to the license terms in the LICENSE file found in the top-level directory
+of this distribution and at http://opencv.org/license.html.
+*/
+
+#ifndef __OPENCV_FACE_ALIGNMENT_HPP__
+#define __OPENCV_FACE_ALIGNMENT_HPP__
+
+#include "opencv2/face/facemark.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/objdetect.hpp"
+#include "opencv2/imgproc.hpp"
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <vector>
+#include <fstream>
+#include <queue>
+#include <algorithm>
+#include <fstream>
```

Comment:
This is a public header. And I believe this huge list of headers are not required in the user application.